### DISCUSSION TITLE
Pause Wallpaper Application exception/white list?

### DISCUSSION THREAD

**Discussion by iSupremacy** [rocksdanister/lively#2210] [Ideas]
I know there's the whitelist to pause but would a whitelist to NOT pause be possible for things like explorer.exe, and windows' setting application  (even Lively itself) to not pause wallpapers?

**Comment by rocksdanister** [comment]
In the settings folder:
https://github.com/rocksdanister/lively/wiki/Getting-Started#storage

There is `AppRules.json` file you can edit:
```json
[
  {
    "AppName": "Discord",
    "Rule": 1
  },
  {
    "AppName": "Foobar",
    "Rule": 0
  }
]
```

1 is for pause and 0 for play, after editing restart Lively.
